checkov/common/output/cyclonedx.py

```diff
@@ -91,9 +60,27 @@ def create_bom(self) -> Bom:
         bom.metadata.tools.add(this_tool)
 
         for report in self.reports:
-            if report.check_type == CheckType.SCA_PACKAGE and self.export_iac_only:
+            if report.check_type in SCA_CHECKTYPES and self.export_iac_only:
                 continue
 
+            if report.check_type == CheckType.SCA_IMAGE:
+                image_record = next(itertools.chain(report.failed_checks, report.passed_checks, report.skipped_checks))
```

Comment:
wouldn't this only add one image to the report, even when you can several different ones in the report?

Reply:
Good catch, fixed